### ISSUE TITLE
Homeserver login error "Error creating SSL context()"

### ISSUE THREAD

**Issue by jrmagots** [mujx/nheko#332]
### System:
- Nheko version: I can't find the settings from login page ( found version 0.4.2.0 on properties version on exe file
- Installation method: I just download the Win64 version and installed on my computer
- Operating System: Windows 10 Build 17134.48

### Actual behavior
When I try to login on my homeserver using nheko, I got the error " **Error creating SSL context ()** "
I can normally logon on my homeserver using riot client.

![image](https://user-images.githubusercontent.com/1504200/40612295-4848f92e-6250-11e8-91fb-a95c65e1ab17.png)

**Comment by mujx** [comment]
Sorry for the late response. Is this happening only on this homeserver? Does matrix.org or disroot.org have the same issue?

**Comment by se17p** [comment]
Hi,
I have the same issue on both 0.4.3 and nightly. Same for disroot and matrix home servers, also for registration. The log contains following output after the app start:

```
    31.769: [warning] - QSslSocket: cannot resolve CRYPTO_num_locks
    31.769: [warning] - QSslSocket: cannot resolve CRYPTO_set_id_callback
    31.769: [warning] - QSslSocket: cannot resolve CRYPTO_set_locking_callback
    31.769: [warning] - QSslSocket: cannot resolve ERR_free_strings
    31.770: [warning] - QSslSocket: cannot resolve EVP_CIPHER_CTX_cleanup
    31.770: [warning] - QSslSocket: cannot resolve EVP_CIPHER_CTX_init
    31.770: [warning] - QSslSocket: cannot resolve sk_new_null
    31.770: [warning] - QSslSocket: cannot resolve sk_push
    31.770: [warning] - QSslSocket: cannot resolve sk_free
    31.770: [warning] - QSslSocket: cannot resolve sk_num
    31.770: [warning] - QSslSocket: cannot resolve sk_pop_free
    31.770: [warning] - QSslSocket: cannot resolve sk_value
    31.770: [warning] - QSslSocket: cannot resolve SSL_library_init
    31.770: [warning] - QSslSocket: cannot resolve SSL_load_error_strings
    31.770: [warning] - QSslSocket: cannot resolve SSL_get_ex_new_index
    31.770: [warning] - QSslSocket: cannot resolve SSLv3_client_method
    31.770: [warning] - QSslSocket: cannot resolve SSLv23_client_method
    31.770: [warning] - QSslSocket: cannot resolve SSLv3_server_method
    31.770: [warning] - QSslSocket: cannot resolve SSLv23_server_method
    31.770: [warning] - QSslSocket: cannot resolve X509_STORE_CTX_get_chain
    31.774: [warning] - QSslSocket: cannot resolve OPENSSL_add_all_algorithms_noconf
    31.774: [warning] - QSslSocket: cannot resolve OPENSSL_add_all_algorithms_conf
    31.774: [warning] - QSslSocket: cannot resolve SSLeay
    31.774: [warning] - Incompatible version of OpenSSL
    31.831: [warning] - QSslSocket: cannot call unresolved function SSLv23_client_method
    31.831: [warning] - QSslSocket: cannot call unresolved function SSL_CTX_new
    31.831: [warning] - QSslSocket: cannot call unresolved function SSL_library_init
    31.831: [warning] - QSslSocket: cannot call unresolved function ERR_get_error
    31.832: [warning] - QSslSocket: cannot call unresolved function ERR_get_error
    44.955: [warning] - QSslSocket: cannot call unresolved function SSLv23_client_method
    44.955: [warning] - QSslSocket: cannot call unresolved function SSL_CTX_new
    44.955: [warning] - QSslSocket: cannot call unresolved function SSL_library_init
    44.955: [warning] - QSslSocket: cannot call unresolved function ERR_get_error
    44.956: [warning] - QSslSocket: cannot call unresolved function ERR_get_error
```

PS. Debian 9 stable.

**Comment by se17p** [comment]
OpenSSL 1.1.0f

**Comment by mujx** [comment]
@se17p Your issue is the same with #156

**Comment by mujx** [comment]
@jrmagots Is this still an issue (with the current nightly release)?

**Comment by kskarthik** [comment]
I also have the same issue with disroot

**Comment by mujx** [comment]
@kskarthik What version are you using?

**Comment by kskarthik** [comment]
latest stable. Nighty has no issues

**Comment by mujx** [comment]
Closing as it is already resolved in the current master and it will be included in the next stable relase (v0.5.0)
